_episodes/04-recipe.md

```diff
@@ -94,7 +101,7 @@ the name: recipe_example.yml LINK
 >  1    # ESMValTool
 >  2    # recipe_example.yml
 >  3    ---
->  4    documentation:
+>  4    documentation
```

Comment:
Why did you delete the colon? I think it should stay if I look at all the recipes...